newbank/server/Customer.java

```diff
@@ -116,7 +116,6 @@ public boolean passwordCorrect(String pw) {
 	 * This method is used to modify the a given balance by a given amount
 	 * @param amount the amount to be transferred
 	 * @param accountName the account to pay from
-	 * @return boolean
```

Comment:
Undo this deletion

Reply:
This deletion was actually intentional. This relates to the modifyAccountBalance function that is void and so doesn't return anything.